src/format.js

```diff
@@ -96,6 +97,17 @@ export function formatRelative(config, state, value, options = {}) {
     let defaults        = format && getNamedFormat(formats, 'relative', format);
     let filteredOptions = filterProps(options, RELATIVE_FORMAT_OPTIONS, defaults);
 
+    // Capture the current threshold values, then temporarily override them with
+    // specific values just for this render.
+    const thresholds = {...IntlRelativeFormat.thresholds};
+    Object.assign(IntlRelativeFormat.thresholds, {
+        second: 60, // seconds to minute
+        minute: 60, // minutes to hour
+        hour  : 24, // hours to day
+        day   : 30, // days to month
```

Comment:
I'll have to look again, but we took the original thresholds from Moment :) and it has `26` for days-to-month.